src/main/java/graphql/ExecutionInput.java

```diff
@@ -306,11 +332,18 @@ public Builder graphQLContext(Map<?, Object> mapOfContext) {
         }
 
         // hidden on purpose
-        private Builder transfer(GraphQLContext graphQLContext) {
+        private Builder internalTransferContext(GraphQLContext graphQLContext) {
```

Comment:
I can rename because its private method.  This is a transferring of the context to a new instance